### DISCUSSION TITLE
java.lang.reflect.InaccessibleObjectException

### DISCUSSION THREAD

**Discussion by utnFBlasco** [dds-jv/foro#320] [Consultas de herramientas]
Buenas al levantar el server me tira el error del título en el spark framework debug, busque en google y se refiere a todos errores de Java 9 en diferentes escenarios y dice que es causado por Java Plataform Module System. HELP ME!!!

![WhatsApp Image 2021-10-27 at 15 38 54](https://user-images.githubusercontent.com/82328433/139128191-b5a2271e-0f8e-416c-9e79-74166a1301ad.jpeg)

![WhatsApp Image 2021-10-27 at 15 48 27](https://user-images.githubusercontent.com/82328433/139128224-5b496f25-6950-4c8f-97cf-a5191f097fd9.jpeg)

**Comment by gastonprieto** [comment]
Hola @utnFBlasco 

Me suena que es la versión de Java, pero no estoy seguro. Si queres copianos el stack en vez de poner el print, y un link al repo donde este esto subido, así vemos que puede ser.
